html/assignment-3/dev-abhishek.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Form</title>
+</head>
+<body>
+  <form>
+    <fieldset>
+      <legend>Personal Information</legend>
+      Salutation:<br><select>
+        <option value="none">none</option>
```

Comment:
replace `none` with `--None--`

please check the issue description to make sure all options are the same
https://github.com/fellow-developers/frontend-practice/issues/10 

```html
<option>--None--</option>
<option>Mr.</option>
<option>Ms.</option>
<option>Mrs.</option>
<option>Dr.</option>
<option>Prof.</option>
```